infra/stream/test/TestFormatter.cpp

```diff
@@ -0,0 +1,241 @@
+#include "infra/stream/Formatter.hpp"
```

Comment:
In general: try to copy the test naming convention we use throughout the codebase. I.e. lower_case_with_underscores_ExceptIdentifiers.